criu/config.c

```diff
@@ -703,6 +703,7 @@ int parse_options(int argc, char **argv, bool *usage_error, bool *has_exec_cmd,
 		BOOL_OPT("mntns-compat-mode", &opts.mntns_compat_mode),
 		BOOL_OPT("unprivileged", &opts.unprivileged),
 		BOOL_OPT("ghost-fiemap", &opts.ghost_fiemap),
+		BOOL_OPT("no-background-fetch", &opts.no_background_fetch),
```

Comment:
@liusdu What is the use-case for this option?

cc: @rppt

Reply:
This is for the test case. We know that CRIU can set up pages in the background or on demand in lazy-page mode. We must disable the background mode to ensure that both the child and parent processes trigger two page faults (#PF). Otherwise, by the time the child and parent processes start running, the lazy page server may have already completed its tasks and exited

Reply:
Ah, I see. We typically add command-line options to enable new CRIU features and document these options in the man page and `criu --help`. However, having a large number of options may lead to confusion for users who are not familiar with the available CRIU features.

Would it be possible to use an environment variable instead? For example, something like the following:
```
CRIU_DISABLE_BACKGROUND_LAZY_PAGE_FETCH=1  criu restore ...
```

Reply:
ok~

Reply:
> Ah, I see. We typically add command-line options to enable new CRIU features and document these options in the man page and `criu --help`. However, having a large number of options may lead to confusion for users who are not familiar with the available CRIU features.
> 
> Would it be possible to use an environment variable instead? For example, something like the following:
> 
> ```
> CRIU_DISABLE_BACKGROUND_LAZY_PAGE_FETCH=1  criu restore ...
> ```

Or maybe even fault injection mechanism, because normally we don't want to disable the background fetch